Kinvey/Kinvey/Client.swift

```diff
@@ -96,7 +96,7 @@ open class Client: Credential {
     open var clientAppVersion: String?
     
     /// Custom request properties for this client instance.
-    open var customRequestProperties: [String : String] = [:]
+    open var customRequestProperties: [String : Any]?
```

Comment:
@heyzooi if the property is called `requestConfiguration` in other places, it should be called something similar here as well. Also worth discussing - if individual methods have the requestConfig option, is there any reason to expose a global config? Overriding, precedence etc adds a lot of (potentially unnecessary) complexity.